gameobjects/settingsscreencontroller.cpp

```diff
@@ -14,17 +14,22 @@ void SettingsScreenController::OnStart()
 {
     mBackButton = Pong::FindGameObjectByName("BackButton")->GetComponent<Button>();
     mBackButton->AddListener(ButtonEvent::Pressed, []() {
-        Pong::LoadSceneNext(SceneType::Title);
+        Pong::LoadSceneNext("Title");
```

Comment:
Any reason you switched from an enum to a string here? (no preference, just curious)

Reply:
Yeah, this way you can load the scene by the name in the json file instead of hardcoding an enum for each scene.